lib/modules/showImages.js

```diff
@@ -1582,7 +1583,7 @@ function trackMediaLoad(link, thing) {
 	}
 }
 
-function setMediaControls(media, lookupUrl, downloadUrl) {
+function setMediaControls(media, lookupUrl, downloadUrl, title) {
```

Comment:
I'd rather avoid propagate `title` as a `Media` instance can be reused by other expandos. It would be easier to retrieve `thing` by `Thing.from(media)` along with the other logic below.

Reply:
Oy vey, I should've check out the Thing class first. Thanks! Apparently that was actually causing the flow errors.